src/App.tsx

```diff
@@ -1,27 +1,60 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [availableGoods, setAvailableGoods] = useState<Good[]>([]);
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+  const showAllClick = async () => {
+    const allGoods = await getAll()
 
-    <button type="button" data-cy="first-five-button">
-      Load 5 first goods
-    </button>
+    setAvailableGoods(allGoods);
+  }
 
-    <button type="button" data-cy="red-button">
-      Load red goods
-    </button>
+  const showFirstFiveSortedGoods = async () => {
+    const firstFive = await get5First()
```

Comment:
What will we do if `get5First` return an error?

Reply:
@dffUqp I hope, I got it right. If you can, give me feedback please))) Thank you!